poetry/core/semver/__init__.py

```diff
@@ -1,151 +0,0 @@
-import re
-
-from typing import Union
-
-from .empty_constraint import EmptyConstraint
-from .exceptions import ParseConstraintError
-from .patterns import BASIC_CONSTRAINT
-from .patterns import CARET_CONSTRAINT
-from .patterns import TILDE_CONSTRAINT
-from .patterns import TILDE_PEP440_CONSTRAINT
-from .patterns import X_CONSTRAINT
-from .version import Version
-from .version_constraint import VersionConstraint
```

Comment:
This removal breaks https://github.com/python-poetry/poetry/blob/6eaa2984d9fb51a837c4ab11643f4dd9d7828d4d/poetry/repositories/repository.py#L5 (current master).